tests/functional/adapter/test_copy_uppercase.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+from conftest import _profile_ci_azure_auto, _profile_user, _profile_user_azure
+from dbt.tests.adapter.simple_copy.fixtures import (
+    _MODELS__ADVANCED_INCREMENTAL,
+    _MODELS__COMPOUND_SORT,
+    _MODELS__DISABLED,
+    _MODELS__EMPTY,
+    _MODELS__INCREMENTAL,
+    _MODELS__INTERLEAVED_SORT,
+    _MODELS__MATERIALIZED,
+    _MODELS__VIEW_MODEL,
+    _MODELS_GET_AND_REF_UPPERCASE,
+    _PROPERTIES__SCHEMA_YML,
+    _SEEDS__SEED_INITIAL,
+)
+from dbt.tests.util import check_relations_equal, run_dbt
+
+
+def dbt_profile_target(request):
```

Comment:
@nszoni are lines 19-27 needed? seems the same (nothing overwritten) as in `tests/conftest.py`?
besides no `pytest.fixture(scope="class")` and no `raise ValueError...`

Reply:
removed 🥇